bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -130,14 +129,15 @@ else:
 
 sciSegs = segmentlistdict(sciSegs)
 if onSrc is None:
-    plot_met = make_grb_segments_plot(wflow, sciSegs, triggertime, triggername,
-                                      seg_dir, fail_criterion=offSrc)
+    plot_met = make_grb_segments_plot(
+        wflow, segmentlistdict(), triggertime, triggername,
+        seg_dir, fail_criterion=offSrc)
```

Comment:
How can it make the plot without the segments??

Reply:
Ah, good catch.  This was intended to be `segmentlistdict(sciSegs)` to make it uniform with lines 117 and 112.
To answer @spxiwh's question, this `segmentlistdict()` is actually needed on line 112 when no data is available.  I'm attaching an example (ignore the GRB name).  The plot will show you the onsource interval and the trigger-time, overlayed to emptiness. 
![GRB190605974_segments-1](https://github.com/user-attachments/assets/110b2c6a-0c21-43d3-92ce-b90d747f60dd)